tests/api/garage/allCars.spec.js

```diff
@@ -0,0 +1,36 @@
+
+import { creatNewUser as test, expect} from "../../../src/fixtures/createNewUser.js";
+
+test.describe.only("Get cars", ()=> {
+    
+    // test.afterEach(async ({ userControllers })=>{
+    //     const deleteUser = await userControllers.deleteUser()
+    //     const deleteUserBody = await deleteUser.json()
+    //     console.log('Delete User:', deleteUserBody);
+
+    //   })
+      
+      test('Get all cars', async({newCar, carsController})=>{
+
+        const response = await carsController.getCars()
+        const resultBody = await response.json()
+        
+        console.log('newCar:', newCar);
+        
+        expect(resultBody).toEqual({
```

Comment:
I would create more than 1 car to tests this endpoint

create them right in the test